introspection/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Facebook, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+include_directories(${CMAKE_SOURCE_DIR}/src/cc)
+include_directories(${CMAKE_SOURCE_DIR}/src/cc/api)
+
+option(INSTALL_INTROSPECTION "Install BPF introspection tools. Those binaries are statically linked and can take plenty of disk space" ON)
```

Comment:
It is static linking. However, all llvm related functions are not called and hence not included in binary. So currently, the size is actually fair small.

-bash-4.3$ ls -l /usr/share/bcc/introspection/bps 
-rwxr-xr-x. 1 root root 37872 Oct 21 03:44 /usr/share/bcc/introspection/bps
-bash-4.3$ 

So I suggest to remove the sentence "Those binaries are statically linked and can take plenty of space."